.github/workflows/docs.yml

```
@@ -5,6 +5,9 @@ on:
     branches:
       - master
       - main
+  pull_request:
+    branches:
+      - *
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 10: did not find expected alphabetic or numeric character

(syntax-check)